<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>955</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>955</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>955</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background-color: rgb(31,31,31);
}
QWidget{
background-color: rgb(31,31,31);
}
QToolButton{
color: rgb(184,191,200);
border: 1px solid rgb(184,191,200);
}
QToolButton:hover
{
border: 2px solid red;
background-color: rgb(103, 104, 114);
}
QToolButton:pressed
{
background-color: rgb(35,190,212);
color: rgb(255, 255, 255);
}
QLineEdit{
border: 2px solid rgb(184,191,200);
color: rgb(184,191,200);
font: 14pt &quot;맑은 고딕&quot;;

}
QLabel{
color: rgb(184,191,200);
font: 14pt &quot;맑은 고딕&quot;;
QLabel::setAlignment (Qt::AlignVCenter );
QLabel::setAlignment (Qt::AlignHCenter );
}
QPushButton{
color: rgb(184,191,200);
background-color: rgb(10, 21, 49);
font: 14pt &quot;맑은 고딕&quot;;
border: 1px solid rgb(184,191,200);
border-radius: 3px;
}
QPushButton:hover
{
border: 2px solid red;
background-color: rgb(103, 104, 114);
}
QPushButton:pressed
{
background-color: rgb(35,190,212);
color: rgb(255, 255, 255);
}

QProgressBar{
border: 1px solid rgb(184,191,200);
color:rgb(200,200,200);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
text-align: center;
}
QProgressBar:chunk {
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}

    QScrollBar:horizontal {
        min-width: 240px;
        height: 13px;
    }

    QScrollBar:vertical {
     min-height: 240px;
     width: 13px;
}

QScrollBar::groove {
    background: gray;
    border-radius: 5px;
}

QScrollBar::handle {
    background: rgb(31,31,31);
	border: 1px solid rgb(184,191,200);
    border-radius: 5px;
}
 QScrollBar::handle:horizontal {
     width: 25px;
}

QScrollBar::handle:vertical {
    height: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="panoToolButton">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(35, 190, 212);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>Pano</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cephToolButton">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Ceph</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
